notifications_api_common/management/commands/register_kanalen.py

```diff
@@ -120,18 +144,25 @@ def handle(self, **options):
                 "`notifications_api_service` configured"
             )
 
-        service = config.notifications_api_service
-
         # use CLI arg or fall back to setting
         kanalen = options["kanalen"] or sorted(
             [kanaal.label for kanaal in KANAAL_REGISTRY]
         )
 
+        client = NotificationsConfig.get_client()
+        assert client
```

Comment:
It was indeed a typeguard, since it checks if a service is configured at the top of `handle` (though I will raise a CommandError there, because currently it doesn't)